template/index.html

```diff
@@ -60,7 +60,7 @@
     <link rel="stylesheet" href="css/utilities.css" />
   </head>
   <body>
-    <div id="hero">
+    <header id="hero"></header>
```

Comment:
This closing tag should not be on the same line. It should replace the `div` element in line `115`

The header contains everything in the hero section. The navbar, the user image, the short bio, and the button. 

Replace the div in line `115` with the closing header.